doc/adr/template.md

```diff
@@ -0,0 +1,72 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+
+Technical Story: [description | ticket/issue URL] <!-- optional -->
+
+## Context and Problem Statement
+
+[Describe the context and problem statement, e.g., in free form using two to three sentences. You may want to articulate the problem in form of a question.]
```

Comment:
I wonder if we should somehow enforce hard-wrapping lines. Thing is, new ADRs are presented in the form of a PR and there you mostly review the files through a diff view.

Reply:
Hmm, but the files wont be touched after the PR is merged. So it would only help during the PR reviewing phase, and there the whole document is completely new anyways...

Reply:
Well, I already have the problem atm reading this line, imagine an entire paragraph. Diff viewers don't soft wrap lines.

Reply:
Maybe prettier is a solution? Seems they are thinking about this problem specifically in the case of MarkDown
https://prettier.io/docs/en/options.html#prose-wrap

Reply:
I guess it does not really hurt, but how to enforce it? We can just promote it and recommend tools. But that would be ok for me